examples/plotting/spans_and_strips.py

```diff
@@ -1,3 +1,11 @@
+''' This example demonstrates the use of ``hspan``, ``wspan``, and ``vstrip``.
```

Comment:
still a typo here "wspan"

Reply:
Oh my, I was literally reading it out loud when I was making the changes, I guess my hands was faster than my mouth. I will effect the changes.